src/ts/client/grapesjs/index.ts

```diff
@@ -289,6 +305,13 @@ export function getEditorConfig(config: ClientConfig): EditorConfig {
       [filterStyles]: {
         appendBefore: '.gjs-sm-sectors',
       },
+      [internalLinksPlugin.toString()]: {
```

Comment:
i don't understand, why did you have to add this? it wasn't already there?

Reply:
It is the result of a merge, it was indeed already there 👍🏼